tendrl/commons/utils/log_utils.py

```diff
@@ -0,0 +1,26 @@
+"""
+Handles logging functionality.
+"""
+import sys
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+
+
+def log(log_priority, publisher_id, msg):
+    """
+    Function used for logging errors/output/info.
+    Args:
+        log_priority : Priority of the Log Message
+        publisher_id : Id of publisher
+        msg          : Message to be logged
+    """
+    try:
+        Event(
+            Message(
+                priority=log_priority,
+                publisher=publisher_id,
+                payload={"message": msg}
+            )
+        )
+    except KeyError:
+            sys.stdout.write(msg)
```

Comment:
We write to stdout and stderr both based on the priority. U will have to keep a check which logs to stderr if the priority is error and otherwise to stdout.